openupgrade_scripts/scripts/l10n_fr/18.0.2.1/pre-migration.py

```diff
@@ -3,682 +3,21 @@
-    ("l10n_fr.tax_report_Z1", "l10n_fr_account.tax_report_Z1"),
-    ("l10n_fr.tax_report_Z2", "l10n_fr_account.tax_report_Z2"),
-    ("l10n_fr.tax_report_Z3", "l10n_fr_account.tax_report_Z3"),
-    ("l10n_fr.tax_report_Z4", "l10n_fr_account.tax_report_Z4"),
-    ("l10n_fr.tax_report_Z5", "l10n_fr_account.tax_report_Z5"),
-    ("l10n_fr.tax_report_credit", "l10n_fr_account.tax_report_credit"),
-    (
-        "l10n_fr.tax_report_credit_constate",
-        "l10n_fr_account.tax_report_credit_constate",
-    ),
-    ("l10n_fr.tax_report_credit_impute", "l10n_fr_account.tax_report_credit_impute"),
-    ("l10n_fr.tax_report_decompte_tva", "l10n_fr_account.tax_report_decompte_tva"),
-    ("l10n_fr.tax_report_determination", "l10n_fr_account.tax_report_determination"),
-    (
-        "l10n_fr.tax_report_montant_op_realisees",
-        "l10n_fr_account.tax_report_montant_op_realisees",
-    ),
-    (
-        "l10n_fr.tax_report_op_imposables_ht",
-        "l10n_fr_account.tax_report_op_imposables_ht",
-    ),
-    (
-        "l10n_fr.tax_report_op_non_imposables",
-        "l10n_fr_account.tax_report_op_non_imposables",
-    ),
-    ("l10n_fr.tax_report_regularisation", "l10n_fr_account.tax_report_regularisation"),
-    ("l10n_fr.tax_report_reliquat", "l10n_fr_account.tax_report_reliquat"),
-    ("l10n_fr.tax_report_tic_tax", "l10n_fr_account.tax_report_tic_tax"),
-    ("l10n_fr.tax_report_tva_brute", "l10n_fr_account.tax_report_tva_brute"),
-    (
-        "l10n_fr.tax_report_tva_brute_autre",
-        "l10n_fr_account.tax_report_tva_brute_autre",
-    ),
-    ("l10n_fr.tax_report_tva_brute_dom", "l10n_fr_account.tax_report_tva_brute_dom"),
-    (
-        "l10n_fr.tax_report_tva_brute_import",
-        "l10n_fr_account.tax_report_tva_brute_import",
-    ),
-    (
-        "l10n_fr.tax_report_tva_brute_metropo",
-        "l10n_fr_account.tax_report_tva_brute_metropo",
-    ),
-    (
-        "l10n_fr.tax_report_tva_brute_petrolier",
-        "l10n_fr_account.tax_report_tva_brute_petrolier",
-    ),
-    ("l10n_fr.tax_report_tva_deductible", "l10n_fr_account.tax_report_tva_deductible"),
-    (
-        "l10n_fr.display_name_in_footer_param",
-        "l10n_fr_account.display_name_in_footer_param",
-    ),
-    (
-        "l10n_fr.account_reports_fr_statements_menu",
-        "l10n_fr_account.account_reports_fr_statements_menu",
-    ),
-]
-
 
 @openupgrade.migrate()
 def migrate(env, version):
-    openupgrade.rename_xmlids(env.cr, _xmlids_renames)
-    openupgrade.logged_query(  # to update 2 problematic tax accounts in l10n_fr_account
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE ir_model_data
+        SET module = 'l10n_fr_account'
+        WHERE module = 'l10n_fr' AND model IN (
+            'account.account.tag', 'account.report', 'account.report.column',
+            'account.report.expression', 'account.report.line',
+            'ir.config_parameter', 'res_bank')
```

Comment:
res.bank

Reply:
Thanks for spotting that, I just created another PR for this typo  : https://github.com/OCA/OpenUpgrade/pull/5290